tests/internal/test_rag.py

```diff
@@ -7,12 +7,12 @@
     test_prompt,
     test_query,
     test_response,
-    trustworthy_rag,  # noqa: F401
-    trustworthy_rag_api_key,  # noqa: F401
```

Comment:
Why did you remove these? These fixtures are not defined in conftest.py, so they need to be imported.

Reply:
Why is this file being updated at all?

Reply:
This was with hatch automatic format fix, I'll look into these